From 99b3c7b6a1b090651ec18248940071997d8d5529 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Tue, 29 Apr 2014 15:39:59 +0200
Subject: [PATCH] Include glib-compat.h in spice-option.c

Commit 8c89485 added a call to g_clear_pointer().
---
 gtk/spice-option.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gtk/spice-option.c b/gtk/spice-option.c
index 7fa3bb5..978953b 100644
--- a/gtk/spice-option.c
+++ b/gtk/spice-option.c
@@ -22,6 +22,7 @@
 #include <stdlib.h>
 #include <glib-object.h>
 #include <glib/gi18n.h>
+#include "glib-compat.h"
 #include "spice-session.h"
 #include "spice-util.h"
 #include "spice-channel-priv.h"
